setup.cfg

```diff
@@ -28,8 +28,10 @@ dev =
   tox
   twisted
   aiounittest
+  mock
```

Comment:
This doesn't seem to use `mock`?

Reply:
It does here:

https://github.com/matrix-org/synapse-3pid-checker/blob/fb066417d7c7fd0336b8dde0c4a49057ce0d02f5/tests/__init__.py#L56-L60

Reply:
Ah, so it does! 👍 Can we use it from `unittest` instead of the third party package?

Reply:
Ah, I always get both confused. I guess the cookiecutter template should be updated then.